tests/metrics.py

```diff
@@ -100,7 +99,7 @@ def test_initial_opt_in_accept():
         # https://openedx.atlassian.net/wiki/spaces/AC/pages/2720432206/Devstack+Metrics
         #
         # Additional metrics require approval (as do changes to
-        # existing ones).
+        # existing ones). Update the list of metrics displayed during opt-in in send-metrics.py
```

Comment:
Maybe this phrasing instead, though?  I want to make sure it's clear that it's an *additional* requirement.
```suggestion
        # existing ones). Changes to metrics also require an update to the
        # list of metrics displayed during opt-in.
```